### PR TITLE
chore: [FRONT-97] Dialog component padding space updated

### PR DESCRIPTION
**Pull request by umrzoq-toshkentov**
**PR description:**

**Implemented:**

Dialog Component DialogActions padding space change

**Checklist:**

- [x] I ran this code locally
- [x] I wrote the necessary tests
- [x] My code follows the [style guidelines](http://bit.ly/sd-web-style-guide)
- [x] I followed the [instructions](http://bit.ly/sd-web-pr) to create a pull request

**JIRA card:**

https://superdispatch.atlassian.net/browse/FRONT-97

**Should know about:**

image

### CODE REVIEW

**Review by doniyor2109 — APPROVED** [review]

File: packages/ui/src/dialog/DialogOverrides.ts
Comment (timurhamitov) [inline packages/ui/src/dialog/DialogOverrides.ts:1]: @umrzoq-toshkentov Are we sure that these changes are necessary?

Comment (umrzoq-toshkentov) [reply packages/ui/src/dialog/DialogOverrides.ts:1]: @timurhamitov yes it is necessary, because I have seen there is update with DialogAction spacing.

Comment (timurhamitov) [reply packages/ui/src/dialog/DialogOverrides.ts:1]: @umrzoq-toshkentov I'm just worried what will happen with existing dialogs. Not all of them are have same structure that we have in examples in storyboard.
And was designers asked for this change? I remember Madina and Artyom said that we need to change only colors for now. Should we have discussion about this to be sure?

Comment (umrzoq-toshkentov) [reply packages/ui/src/dialog/DialogOverrides.ts:1]: yeap I got it. But there is no any breaking changes. If you want I can revert it

**Pull request merged**